src/cryptsetup/cryptsetup-token-systemd-tpm2/cryptsetup-token-systemd-tpm2.c

```diff
@@ -0,0 +1,223 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <errno.h>
+#include <libcryptsetup.h>
+
+#include "cryptsetup-token.h"
+#include "hexdecoct.h"
+#include "memory-util.h"
+#include "tpm2-util.h"
+#include "luks2-tpm2.h"
+
+#define TOKEN_NAME "systemd-tpm2"
+#define TOKEN_VERSION_MAJOR "1"
+#define TOKEN_VERSION_MINOR "0"
+
+#define crypt_log_dbg(cd, x...) crypt_logf(cd, CRYPT_LOG_DEBUG, x)
```

Comment:
hmm we usually prefer standard syntax over gcc-specific syntax if both are equivalent and not a lot uglier. hence maybe:

```c
#define crypt_log_dbg(cd, ...) crypt_logf(cd, CRYPT_LOG_DEBUG, __VA_ARGS__)
```